### DISCUSSION TITLE
Can't the `wezterm` package on Archlinux get rid of font deps?

### DISCUSSION THREAD

**Discussion by vollowx** [wezterm/wezterm#4734] [General]
![image](https://github.com/wez/wezterm/assets/73375859/7f18cba3-e13b-4eda-b97f-66881d2066e6)

I personally don't use them, and the extra 20mb is really annoying

**Comment by nekowinston** [comment]
I think you should report this over at https://gitlab.archlinux.org/archlinux/packaging/packages/wezterm/-/issues, since how distribution is handled by third parties isn't really a WezTerm issue.

**Reply by vollowx** [reply]
Sorry for this and surely I will :heart:

**Reply by wez** [reply]
Note that wezterm does require those fonts to run. More context here: https://github.com/wez/wezterm/issues/4689#issuecomment-1867248778

**Reply by vollowx** [reply]
@wez I wonder the reason. Why don't we just set the default to `sans-serif` and `monospace`? Then if the default UI doesn't look normal, the user will try to re-configurate.

**Reply by wez** [reply]
wezterm runs on multiple platforms and I want the out of the box experience to be the same on all of them. In particular, it is not worth (to me) the engineering overheads to special case this, and 20MB of storage on a workstation is cheap enough that I'm not losing sleep over this.

**Reply by vollowx** [reply]
reasonable enough :smile:
